server.js

```diff
@@ -1,44 +1,33 @@
-const http = require("http");
-const host = "localhost";
-const port = 3000;
-
-const requestHandler = (req, res) => {
-  res.end("Hello");
-};
-
-/*
-const requestHandler = (req, res) => {
-  res.setHeader("Content-Type", "application/json");
-  res.writeHead(200);
-  res.end(`{"message": "This is a JSON response"}`);
-};
-
+const http = require('http');
+const url = require('url');
+const airportInfo = require('./src/airportDetailsRetriever');
 
+const host = 'localhost';
+const port = 3000;
 const requestHandler = (req, res) => {
-  res.setHeader("Content-Type", "application/json");
+  res.setHeader('Content-Type', 'application/json');
   switch (req.url) {
     case "/":
       res.writeHead(200);
-      res.end("Its working!");
-      break;
-    case "/ping":
-      res.writeHead(200);
-      res.end("Pong");
+      res.write("Server working!\n");
```

Comment:
This unnecessarily appends "Server Working" to every other endpoint response whether success or error case. Define a specific `/heath` route for checking condition of server. Root level is usually for your app landing page, it's not used as a health check end point